.github/CONTRIBUTING.md

```diff
@@ -119,6 +120,52 @@ yalc push
 
 This is a shortcut for `yalc publish --push`, which "publishes" the package again and updates it in every project that installed it.
 
+### How to add new tokens
+
+To introduce new design tokens to the Orbit design system, we rely on the versatile capabilities of style-dictionary. Whether you're enhancing specific components or introducing global tokens, the process is straightforward and consistent.
+
+Follow these steps to seamlessly integrate new tokens:
+
+Navigate to Token Definitions: Begin by creating or editing a JSON file within the [dictionary/definitions folder](https://github.com/kiwicom/orbit/tree/master/packages/orbit-design-tokens/src/dictionary/definitions). This is where the token definitions reside.
+
+Craft Token Entries: Craft your new token entries within the JSON file. For instance, if you're adding a token to the Button component, structure it like the following example:
+
+```json
+{
+  "component": {
+    "button": {
+      "stone": {
+        "background": {
+          "type": "color",
+          "value": "{foundation.palette.stone.normal}"
+        }
+        // ... other properties
+      }
+    }
+  }
+}
+```
+
+More about token schema you can read in [SCHEMA documentation](https://github.com/kiwicom/orbit/blob/master/packages/orbit-design-tokens/docs/SCHEMA.md)
+
+Refer to the token schema documentation for a comprehensive understanding of token properties.
+
+- Categorize Tokens: Organize your tokens under appropriate categories, either globally or component-specific. This structure ensures clarity and maintainability.
+
+- Generate Tokens: Once your token schema is ready, execute command:
+
+```bash
+yarn tokens build
+```
```

Comment:
Although I don't disagree with what this paragraph says, it feels a bit distracting from the objectiveness that I expect from a technical documentation 😄 I won't block this, at all, but it feels weird to me. Team's decision is fine to me

Reply:
This should be on line 165, the second to last paragraph (weird Github mobile app)

Reply:
> Although I don't disagree with what this paragraph says, it feels a bit distracting from the objectiveness that I expect from a technical documentation 😄 I won't block this, at all, but it feels weird to me. Team's decision is fine to me

Ofc it can be always improved, I am not a professional technial writer and particaulary in that case I prefer to have something done on that front rather do not have anything at all ;)